tekore/_client/api/search.py

```diff
@@ -80,6 +80,12 @@ def search(
             tracks, = spotify.search('monty python')
             artists, = spotify.search('sheeran', types=('artist',))
             albums, tracks = spotify.search('piano', types=('album', 'track'))
+            spotify.search('gold album:boba artist:abba', types=('track',))
+            spotify.search('bob year:1980-2020', types=('show',))
+
+        .. note::
+            You can narrow down search results by specifying filed filters (e.g. year range, genre).
```

Comment:
Also typo here, 'Filed' = 'Field'?

Reply:
Yes, that's a typo. Meant to write "field".